runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/sdk/options/DataflowPipelineOptions.java

```diff
@@ -68,7 +68,9 @@
    */
   @Description("The Dataflow job name is used as an idempotence key within the Dataflow service. "
       + "If there is an existing job that is currently active, another active job with the same "
-      + "name will not be able to be created. Defaults to using the ApplicationName-UserName-Date.")
+      + "name will not be able to be created. Defaults to using the ApplicationName-UserName-Date. "
```

Comment:
If there's an existing and running job with the same name in the GCP project, you won't be able to start a new one unless the new job is an explicit update of the previous one.